guides/v2.2/magento-functional-testing-framework/release-2/test.md

```diff
@@ -73,52 +76,55 @@ There are several XML elements that are used in `<tests>` in the MFTF.
 
 `<tests>` is a container for multiple tests. It is a group of test methods that define test flows within a test case.
 
-`<tests>` must contain at least one [`<test>`](#test-tag).
+`<tests>` must contain at least one [`<test>`].
 
 
 ### test {#test-tag}
 
-`<test>` is a set of steps, including [actions](./test/actions.html) and [assertions](./test/assertions.html). It is a sequence of test steps that define test flow within a test method.
+`<test>` is a set of steps, including [actions] and [assertions][assertion]. It is a sequence of test steps that define test flow within a test method.
 
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`name`|string|optional| A test identifier used while merging.
+`name`|string|optional|The test identifier.
 `remove`|boolean|optional|Set `true` to remove the test when merging.
+`insertBefore`|string|optional| This option is used for [merging]. It enables you to add all test actions contained in the test into test with the same name BEFORE the test step with `stepKey` that you assigned in `insertBefore`.
```

Comment:
the test into test
"the original test into a test"?

Reply:
👍